src/phoneField/__snapshots__/index.unit.tsx.snap

```diff
@@ -0,0 +1,165 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PhoneField should have the expected markup in the DOM with custom settings 1`] = `
+<div
+  aria-labelledby="labelId"
```

Comment:
There's no tag with `id="labelId"` in the snapshot.
If we can output the component without the accessibility string, then we should not add this `aria-labelledby`

Reply:
Ok as seen IRL, the corresponding tag has to be included in the children when using the component.